lib/caqti_request.mli

```diff
@@ -135,16 +134,16 @@ val create_p :
     described in the introduction of this section.
 
     @param oneshot
-      For queries generated on-demand or which are otherwise executed only once,
-      pass [true] do make the query non-prepared.  By default queries are
-      prepared, which is suitable for requests defined at the module level.
+    For queries generated on-demand or which are otherwise executed only once,
+    pass [true] do make the query non-prepared.  By default queries are
+    prepared, which is suitable for requests defined at the module level.
 
     @param env
-      [env driver_info key] shall provide the value to substitute for a
-      reference to [key] in the preliminary query string, or raise [Not_found]
-      to indicate the reference to [key] is invalid.  [Not_found] will be
-      re-raised as [Invalid_argument] with additional information to help locate
-      the bug. *)
+    [env driver_info key] shall provide the value to substitute for a
+    reference to [key] in the preliminary query string, or raise [Not_found]
+    to indicate the reference to [key] is invalid.  [Not_found] will be
+    re-raised as [Invalid_argument] with additional information to help locate
+    the bug. *)
```

Comment:
Please don't remove indentation here.

Reply:
Thanks.  I believe I have removed all of the unintended whitespace changes.